src/main/java/by/custom/utilcalculator/exception/UserFileNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package by.custom.utilcalculator.exception;
+
+import by.custom.utilcalculator.service.BundleResourcesServant;
+
+public class UserFileNotFoundException extends Exception {
+    private final BundleResourcesServant bundle;
+    public UserFileNotFoundException(){
+        bundle = BundleResourcesServant.getInstance();
+    }
+
+    @Override
+    public String getMessage() {
+        return bundle.getString("answers.user_not_found");
```

Comment:
In my view Exceptions should not own bundle 
```suggestion
        return BundleResourcesServant.getInstance().getString("answers.user_not_found");
```

Reply:
I couldn't even imagine this construction. Sure, now we can remove bundle and use new construstion